spec/rubocop/cop/rspec/builtin_matcher_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::BuiltinMatcher do
+  it 'registers an offense when using `be_all`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_all(bar)
+                     ^^^^^^^^^^^ Use `all` instead of `be_all`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to all(bar)
+    RUBY
+  end
+
+  it 'registers an offense when using `be_cover`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_cover(bar)
+                     ^^^^^^^^^^^^^ Use `cover` instead of `be_cover`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to cover(bar)
+    RUBY
+  end
+
+  it 'registers an offense when using `be_end_with`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_end_with(bar)
+                     ^^^^^^^^^^^^^^^^ Use `end_with` instead of `be_end_with`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to end_with(bar)
+    RUBY
+  end
+
+  it 'registers an offense when using `be_eql`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_eql(bar)
+                     ^^^^^^^^^^^ Use `eql` instead of `be_eql`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to eql(bar)
+    RUBY
+  end
+
+  it 'registers an offense when using `be_equal`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_equal(bar)
+                     ^^^^^^^^^^^^^ Use `equal` instead of `be_equal`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to equal(bar)
+    RUBY
+  end
+
+  it 'registers an offense when using `be_exist`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_exist(bar)
+                     ^^^^^^^^^^^^^ Use `exist` instead of `be_exist`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to exist(bar)
+    RUBY
+  end
+
+  it 'registers an offense when using `be_exists`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_exists(bar)
+                     ^^^^^^^^^^^^^^ Use `exist` instead of `be_exists`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to exist(bar)
```

Comment:
Does it accept an argument?

Reply:
It seems accept one argument:
https://rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers#exist-instance_method

```ruby
expect(File).to exist("path/to/file")
```